org.adempiere.base/src/org/compiere/process/SvrProcess.java

```diff
@@ -270,9 +270,9 @@ private boolean process()
 			if (msg == null)
 				msg = e.toString();
 			if (e.getCause() != null)
-				log.log(Level.SEVERE, msg, e.getCause());
+				log.log(Level.SEVERE, Msg.parseTranslation(getCtx(), msg), e.getCause());
 			else 
-				log.log(Level.SEVERE, msg, e);
+				log.log(Level.SEVERE, Msg.parseTranslation(getCtx(), msg), e);
 			success = false;

```

Comment:
this can potentially leads to many "Not found: ..." warnings in server log.

Reply:
> this can potentially leads to many "Not found: ..." warnings in server log.

Hi @hengsin - the Msg.parseTranslation doesn't throw error, if there are no context variables it simply returns the same string